lib/datastore.dart

```diff
@@ -252,8 +252,20 @@ class FilterRelation {
   // ignore: constant_identifier_names
   static const FilterRelation LessThanOrEqual = FilterRelation._('<=');
   // ignore: constant_identifier_names
+  static const FilterRelation GreaterThan = FilterRelation._('>');
+
+  /// Old misspelled name for [GreaterThan], retained for compatibility.
+  ///
+  /// @nodoc
+  @Deprecated('Use FilterRelation.GreaterThan instead')
   static const FilterRelation GreatherThan = FilterRelation._('>');
```

Comment:
```suggestion
  static const FilterRelation GreatherThan = GreaterThan;
```

We want identity equality to be preserved, we want `GreatherThan == GreaterThan`.

The `_relationMapping` stuff  in `lib/src/db/db.dart` won't work.

Reply:
Can you clarify won't work?

Reply:
Sorry, I got it wrong, it was `relationMapping` in `lib/src/datastore_impl.dart`:

https://github.com/dart-lang/gcloud/blob/eee174cdbfaeaef0c977f0b38e26c2d0a85ce4b5/lib/src/datastore_impl.dart#L204-L210

Would cause problems in: 
https://github.com/dart-lang/gcloud/blob/eee174cdbfaeaef0c977f0b38e26c2d0a85ce4b5/lib/src/datastore_impl.dart#L214-L217

Unless, `GreatherThan` and `GreaterThan` are the same object. In practice, I guess it also creates one object less to use the same object, but that's more like a nit :rofl: